CMakeLists.txt

```diff
@@ -324,7 +324,8 @@ write_basic_package_version_file(
 
 install(TARGETS yaml-cpp 
     EXPORT "${targets_export_name}"
-    INCLUDES DESTINATION "${include_install_dir}"
+	INCLUDES DESTINATION "${include_install_dir}"
+	RUNTIME DESTINATION ${LIB_INSTALL_DIR}
```

Comment:
Please use `bin` or introduce `bin_install_dir` variable with `bin` value.

Reply:
Oh, it's already `bin`:
* https://github.com/hunter-packages/yaml-cpp/blob/5ba9e664bc65e4c0e3537d433f33d1fec3bc5e65/CMakeLists.txt#L248-L266

Okay, I see, a little bit confusing.

Reply:
yes, I feel it was already thought for that. But if you prefer to use bin or any other variable, I dont mind.